bigbluebutton-html5/imports/ui/components/modal/base/styles.scss

```diff
@@ -22,9 +22,7 @@
 
 .overlay {
   z-index: 1000;
-  // background: transparentize($color-white, .35);
-  background: #fff;
-  // background: transparentize($color-gray-dark, .40);
+  background: transparentize($color-gray-dark, .40);
```

Comment:
Tyler wants the modals to have white background and the dialogs (audio case) to have a transparent background.